packages/editorconfig/src/node/editorconfig-service-impl.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { injectable, inject } from "inversify";
+import { WorkspaceServer } from "@theia/workspace/lib/common";
+import * as editorconfig from 'editorconfig';
+import { KnownProps } from "editorconfig";
+import { EditorconfigService } from "../common/editorconfig-interface";
+
+@injectable()
+export class EditorconfigServiceImpl implements EditorconfigService {
+
+    constructor(
+        @inject(WorkspaceServer) protected readonly workspaceServer: WorkspaceServer
```

Comment:
Please use property injection, if possible: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#dependency-injection.

Reply:
WorkspaceServer does not used anymore. Thanks